ec2/InstanceType.dhall

```diff
@@ -400,7 +400,13 @@ let alternatives =
                 , permitLosingLocalDisk : Bool
                 , permitCrossInstanceFamily : Bool
                 }
-            , default = {=}
+            , default =
+              { permitLowerClassCPU = False
+              , permitWorseNetwork = False
+              , permitLowerEBSBandwidth = False
+              , permitLosingLocalDisk = False
+              , permitCrossInstanceFamily = False
+              }
```

Comment:
I understand where you're coming from on this (it's the safe default), but I originally intentionally had an empty default because I wanted the downstream user to think about each of these values and for the choice to be distinctly visible. I don't think that there are enough options here or that alternatives are used in enough places for there to be enough value in making the decision for the user instead of asking the user whether or not he wants to make a safe choice.

Reply:
Agree. But I think we will soon need a README file for that